### PR TITLE
Reduce default bins in mt and iso for abcd method

### PR DESCRIPTION
**Pull request by davidwalter2**
Currently, there are several bins in high mT and high Iso that would be needed for the extended 2D ABCD method, but since we are using the extended 1D as default and don't plan to switch to 2D in the near future we can reduce these bins and reduce the histogram size, and with this also resource consumption.

### CODE REVIEW

**Comment by davidwalter2** [comment]
Changes are as expected very small and should be due to numerical differences,
This PR:
https://cmsmwbot.web.cern.ch/WMassAnalysis/PRValidation/PR486/2024_06_28/
Reference:
https://cmsmwbot.web.cern.ch/WMassAnalysis/PRValidation/ReferenceRuns/2024_06_28_0052a83/
From my point of view this is ready to be merged.

File: utilities/common.py [hunk 1]
```diff
@@ -127,14 +128,27 @@ def get_binning_fakes_pt(min_pt, max_pt):
     #edges = np.append(edges, [max_pt])
     return edges
 
-def get_binning_fakes_mt(mt_cut=40):
+
+def get_binning_fakes_mt(mt_cut=40, high_mt_bins=False):
     edges = np.array([0, int(mt_cut/2.), mt_cut])
```
Comment (bendavid) [inline utilities/common.py:133]: What's the reason to even have the bin boundary 20 GeV here?

Comment (cippy) [reply utilities/common.py:133]: I think it's to have the 2 regions at low mt, 0-20 and 20-40

Comment (davidwalter2) [reply utilities/common.py:133]: Right

**Comment by cippy** [comment]
I also think it is ready to merge, whenever you wish

**Pull request merged**